test/evergreen.yml

```diff
@@ -370,10 +370,20 @@ functions:
     - command: perf.send
       params:
         file: ./wiredtiger/cmake_build/test/cppsuite/${test_name}.json
+    # Delete unnecessary data from the upload.
+    - command: shell.exec
+      params:
+        script: |
+          rm -rf wiredtiger/cmake_build/examples
+          rm -rf wiredtiger/cmake_build/bench
+          mv wiredtiger/cmake_build/test/cppsuite wiredtiger/cmake_build/
+          rm -rf wiredtiger/cmake_build/test/
+          mkdir wiredtiger/cmake_build/test/
+          mv wiredtiger/cmake_build/cppsuite wiredtiger/cmake_build/test/cppsuite
```

Comment:
If I understand correctly, the set of commands aims to leave only the contents under `test/cppsuite`, and the change below (line 386) tries to archive the parent folder `wiredtiger/cmake_build` to make the core dump easier to parse after artifacts download? 

I feel the below lines of changes would achieve a safe effect (and hopefully in a cleaner manner). Want to give a try?
```
        source_dir: wiredtiger/cmake_build
        include:
          - test/cppsuite/**"
 ```